### ISSUE TITLE
how to get DAVIS2017m/FlowImages_gap1,Do we need to use Raft to generate it？

### ISSUE THREAD

**Issue by lxin98** [Jyxarthur/flowsam#10]
Great work.how to get DAVIS2017m/FlowImages_gap1,Do we need to use Raft to generate it？

**Comment by Jyxarthur** [comment]
Thanks for your interest in our work. 
Yes, please follow the instructions in `flow` folder, which provides the RAFT code to generate "FlowImages_gap1" results from RGB.

**Comment by lxin98** [comment]
@Jyxarthur Thank you for your reply. May I ask if there are many optical flow data required, such as FlowImages_gap-1, FlowImages_gap-2,FlowImages_gap-3...， I ran raft and generated FlowImages_gap1, FlowImages_gap-1, and Flows_gap1, Flows_gap-1. When running evaluate, it displayed/home/lanxin/Desktop/data/test/FlowImages_gap-2/test/00081.png, can't open/read file: check file path/integrity

**Comment by lxin98** [comment]
I changed -- flow_gaps=1, -1,2, -2 to 1, -1, hoping it works

**Comment by Jyxarthur** [comment]
Yes, ideally generating  `FlowImages_gap1, FlowImages_gap-1, FlowImages_gap2, FlowImages_gap-2` would work as the default, but it is okay to also use `1,-1` (which could lead to some performance variation).
